src/main/starlark/core/repositories/kotlin/capabilities_2.1.bzl.com_github_jetbrains_kotlin.bazel

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 The Bazel Authors. All rights reserved.
```

Comment:
the options are missing from here and it needs to be registered in `src/main/starlark/core/repositories/compiler.bzl`, otherwise all of the 2.1 options won't be usable.

Reply:
Need to finish automating the process. :p